src/components/Label.tsx

```diff
@@ -0,0 +1,47 @@
+import classNames from "classnames";
+import PropTypes from "prop-types";
+import React, { FunctionComponent, ReactNode } from "react";
+
+import "lbh-frontend/lbh/core/_label.scss";
+
+/**
+ * The property types for {@link Label}.
+ *
+ * @param {string} [id]
+ * @param {string} [labelFor]
+ * @param {string} [className]
+ * @param {ReactNode} children The label content
+ */
+export interface LabelProps {
+  id?: string;
+  className?: string;
+  labelFor?: string;
+  children: ReactNode;
+}
+
+/**
+ * A component for providing labels on inputs.
+ *
+ * @param {LabelProps} props
+ */
+export const Label: FunctionComponent<LabelProps> = ({
+  id,
+  labelFor,
+  className,
+  children
+}: LabelProps) => (
```

Comment:
```suggestion
}: LabelProps): ReactElement => (
```

VSCode will suggest this as a quick fix after updating the docstring.